### ISSUE TITLE
GNOME energy settings not saved after reboot

### ISSUE THREAD

**Issue by ploum** [regolith-linux/regolith-desktop#606]
- [ x] I've [run a search on the existing bug reports](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests) to make sure I'm not filing a duplicate (you can use the search box at the top left corner)
- [ x] I've read [the Regolith documentation](https://regolith-linux.org/docs/) and didn't find any relevant topic
- [ ] I tried [asking on Slack](https://regolith-linux.org/interact/) (especially for configuration related issues)

**Describe the bug**

In the GNOME settings -> Energy panel, I choose "economy" (instead of the default "standard" or "performance"). After a reboot, the setting has been reverted.

In fact, it looks like all the settings on the "energy" panel are reverted after a reboot.

**Installation Details**
 - Regolith Install Type: [ISO/PPA] : ISO then upgrade through PPA
 - Regolith Version: [e.g. 1.4] :  PPA stable (so 1.6)
 - PPA url: [default: ppa:regolith-linux/release] stable
 - Host OS (for PPA): [e.g. Ubuntu 20.04]  : Ubuntu 21.04

**Additional context**

Wondering if this could be related to the upgrade to Ubuntu 21.04 as I don’t remember having the bug before (but I don’t remember playing with those settings before)

**Comment by kgilmer** [comment]
Hi @ploum , are you able to reproduce this w/ the gnome-flashback session?  This would indicate an issue with that or something in Regolith itself.

**Comment by ploum** [comment]
indeed, I’m able to reproduce this bug on both gnome-flashback and GNOME.
